modules/cnf-topology-aware-lifecycle-manager-precache-image-filter.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+// Epic CNF-6848 (4.13), Story TELCODOCS-949
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_content-type: CONCEPT
+[id="talo-precache-feature-image-filter_{context}"]
+= Using the container image pre-cache filter
+
+The pre-cache feature typically downloads more images than a cluster needs for an update. You can control which pre-cache images are downloaded to a cluster. This decreases download time, and saves bandwidth and storage.
+
+The following `ConfigMap` example shows how you can exclude images using the `excludePrecachePatterns` field.
```

Comment:
Each item is a basic regular expression pattern. The list of the images will be filtered out using `grep -v`. For example if you add `aws` to the `excludePrecachePatterns`, every image that has `aws` in its name will not be downloaded.